### PR TITLE
Removal of Species locking & Spartan Self Destruct Changes

### PR DESCRIPTION
**Pull request by Stingray540**
:cl: Stingray
tweak: Moves the Elite/Spartan EVA mode to the Abilities tab, along with Spartan Self Destruct for ease of access. The latter also now has a dedicated action button.
tweak: Removes my covenant species lock because XO used cursed halo 5 lore against me.
/:cl:

### CODE REVIEW

File: code/modules/halo/weapons/covenant/launchers.dm [hunk 1]
```diff
@@ -30,7 +30,7 @@
 	wielded_item_state = "fuelrod-wielded"
 	salvage_components = list(/obj/item/plasma_core)
 	matter = list("nanolaminate" = 2, "kemocite" = 1)
-
+	heavy_covenant = 1
```
Comment (bloxgate) [inline code/modules/halo/weapons/covenant/launchers.dm]: Please add a line between this and the next definition

**Review by bloxgate — DISMISSED** [review]
Looks good, aside from one small formatting thing.

**Comment by X0-11** [comment]
The restrictions aren't happening. Thanks for the reminder to strip those out. Rest seems fine

File: code/modules/halo/clothing/spartan_armour.dm [hunk 2]
```diff
@@ -62,6 +64,9 @@
 	totalshields = 100
 	item_state_slots = list(slot_l_hand_str = "syndicate-black", slot_r_hand_str = "syndicate-black")
 
+/obj/item/clothing/suit/armor/special/spartan/ui_action_click()
```
Comment (X0-11) [inline code/modules/halo/clothing/spartan_armour.dm:67]: I think this interferes with armour ability code.
You may have to find a different way to do this.

**Review by X0-11 — CHANGES_REQUESTED** [review]

File: code/modules/halo/weapons/covenant/launchers.dm (hunk 1 above)
Comment (Stingray540) [reply code/modules/halo/weapons/covenant/launchers.dm]: Resolved as it has been removed.

**Review by X0-11 — APPROVED** [review]

**Pull request merged**